backend/src/main/java/org/cryptomator/hub/entities/LegacyDevice.java

```diff
@@ -1,12 +1,30 @@
 package org.cryptomator.hub.entities;
 
 import io.quarkus.hibernate.orm.panache.PanacheRepositoryBase;
+import io.quarkus.panache.common.Parameters;
 import jakarta.enterprise.context.ApplicationScoped;
 import jakarta.persistence.Column;
 import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.FetchType;
 import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.NamedQuery;
 import jakarta.persistence.Table;
 
+import java.time.Instant;
+import java.util.List;
+import java.util.stream.Stream;
+
+@NamedQuery(name = "LegacyDevice.allInList",
+		query = """
+				SELECT d
+				FROM LegacyDevice d
+				WHERE d.id IN :ids
+				""")
+@NamedQuery(name = "LegacyDevice.deleteByOwner", query = "DELETE FROM LegacyDevice d WHERE d.owner.id = :userId")
```

Comment:
for same reason as stated above (if I'm not mistaken):

```suggestion
@NamedQuery(name = "LegacyDevice.deleteByOwner", query = "DELETE FROM LegacyDevice d WHERE d.ownerId = :userId")
```

Reply:
See https://github.com/cryptomator/hub/pull/331#discussion_r1992209635

Reply:
Fixed in https://github.com/cryptomator/hub/commit/e86a22d58bde610d73512257ce9c18c2102cc69a